test/type/types.swift

```diff
@@ -193,7 +193,7 @@ do {
   // expected-error@-1 {{cannot find type 'imAFunction' in scope}}
 }
 
-typealias A = (inout Int ..., Int ... = [42, 12]) -> Void // expected-error {{'inout' must not be used on variadic parameters}}
+typealias A = (inout Int ..., Int ... = [42, 12]) -> Void // expected-error {{inout' must not be used on variadic parameters}}
```

Comment:
Oops - `'` has been removed.

Reply:
Could you please add the following test:

```swift
do {
  struct Test {
    init(_: inout Int...) {}
    func test(_: inout String...) {}
   subscript(_: inout Double...) {}
  }
}
```

To make sure that the compiler produces "'inout' must not be used on variadic parameters" error in these cases too.

Reply:
Thanks for catching these! I've added the tests for checking the inout usages in variadic parameters. For the third case with subscript, the error message will be `'inout' may only be used on function or initializer parameters` as inout itself isn't allowed in subscripts anyways.